node/src/components/consensus/highway_core/state/panorama.rs

```diff
@@ -107,7 +107,12 @@ impl<C: Context> Panorama<C> {
         state: &'a State<C>,
     ) -> impl Iterator<Item = &'a Vote<C>> {
         let to_vote = move |vh: &C::Hash| state.vote(vh);
-        self.iter().filter_map(Observation::correct).map(to_vote)
+        self.iter_correct_hashes().map(to_vote)
+    }
+
+    /// Returns an iterator over all honest validators' latest messages' hashes.
```

Comment:
```suggestion
    /// Returns an iterator over all honest validators' latest votes' hashes.
```
(Also in `iter_correct`… and probably a few other places in the code. :sweat_smile: Would just be good to be consistent.)